app/components/card_component.rb

```diff
@@ -1,36 +1,37 @@
 # frozen_string_literal: true
 
 class CardComponent < ApplicationComponent
-  TYPES = %i[default primary success info warning danger].freeze
+  TYPES = %i[default primary secondary success info warning danger].freeze
 
   renders_one :header
   renders_one :footer
 
   erb_template <<~ERB
-    <div class="panel panel-<%= @type %>">
-
+    <div class="<%= class_names("card mb-3 \#{@extra_classes}", "border-\#{@type}": @type) %>">
```

Comment:
```suggestion
    <div class="<%= class_names("card mb-3", @extra_classes, "border-#{@type}": @type) %>">
```

Is there any reason to have put `\` before `#`?

Reply:
Yes, otherwhile it didn't work, because we are in `erb_template`

Reply:
Ah... yes of course